src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/commands/ImportCommands.kt

```diff
@@ -15,27 +16,42 @@ import java.time.temporal.ChronoUnit
 class ImportCommands(
   @Autowired private val importService: ImportService
 ) {
+
+  private val logger = LoggerFactory.getLogger(javaClass)
```

Comment:
A small refactoring to add some logging.  No functional change to the underlying commands.